lib/hets-agent/application.rb

```diff
@@ -41,6 +47,41 @@ def id
       def normalize_paths
         Settings.hets.path = Pathname.new(Settings.hets.path)
       end
+
+      def initialize_bunny
+        username = Settings.rabbitmq.username
+        password = Settings.rabbitmq.password
+        host = Settings.rabbitmq.host || 'localhost'
+        port = Settings.rabbitmq.port || 5672
+        connection_string = "amqp://#{username}:#{password}@#{host}:#{port}"
+        @bunny = Bunny.new(connection_string)
+      end
+
+      def initialize_version_requirement
+        @hets_version_requirement =
+          HetsAgent::VersionRequirementRetriever.new.call
+
+        return unless hets_version_requirement.nil?
+
+        message =
+          ['No version requirement for Hets received.',
+           'Please start the ontohub-backend and try again.'].join("\n")
+        raise HetsAgent::BootingError, message
+      end
+
+      def initialize_version_available
+        version_request = HetsAgent::Hets::VersionRequest.new
+        @hets_version_available =
+          HetsAgent::Hets::Caller.call(version_request).output
+
+        unless Gem::Requirement.new(hets_version_requirement).
+            satisfied_by?(Gem::Version.new(hets_version_available))
+          message =
+            "The available Hets version #{hets_version_available} does not "\
+            "satisfy the requirement #{hets_version_requirement}."
+          raise HetsAgent::IncompatibleVersionError, message
```

Comment:
This would make multiple backends talking to the same Hets instances difficult to implement, I think (for e.g. load balancing, if we want to keep this usecase in mind).